terasoluna-gfw-parent/pom.xml

```diff
@@ -591,6 +604,9 @@
     <!-- == MyBatis3 == -->
     <org.mybatis.version>3.5.0</org.mybatis.version>
     <org.mybatis.spring.version>2.0.0</org.mybatis.spring.version>
+    <!-- == Hibernate == -->
+    <!--The version was changed independently as a countermeasure against vulnerabilities of CVE-2020-25638.-->
```

Comment:
- [ ] It is good to specify what you changed from.

> Changed from Spring Boot as a countermeasure against CVE-2020-25638.

Reply:
Correspond with 2dd06dd234f7850d9f9840c740a3c2b38e498ec4